dss_datamover/socket_communication.py

```diff
@@ -375,7 +380,9 @@ def recv_json(self, format="JSON"):
         except socket.timeout as e:
             self.logger.error("ServerSocket: Timeout ({} seconds) from recv function".format((datetime.now() - time_started).seconds))
         except Exception as e:
-            self.logger.error(f"ServerSocket: Exception {e}")
+            if (datetime.now() - self.last_exception_log_time).seconds > LOG_INTERVAL:
+                self.logger.error(f"ServerSocket: Exception {e}")
```

Comment:
I think adding host and port would be useful here

Reply:
Yes it will be useful, but the port and host are not stored as member, it's hard to fetch here.